web/src/features/hfiCalculator/slices/hfiCalculatorSlice.ts

```diff
@@ -227,6 +249,17 @@ export const fetchGetPrepDateRange =
     }
   }
 
+export const fetchFuelTypes = (): AppThunk => async dispatch => {
+  try {
+    dispatch(fetchFuelTypesStart())
+    const fuelTypes = await getFuelTypes()
+    dispatch(setFuelTypes(fuelTypes))
```

Comment:
Why is this not being covered by tests?

Reply:
I was leaning heavily into Agile Principle #10?
(Actually, I was just too focussed on updating the existing unit tests and forgot to write any new ones for this new call)